index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Push-Up-Master</title>
+  </head>
+
+  <body>
+    <div class="container">
+      <div>
+        <div id="display" class="remaining">00:00</div>
+      </div>
+      <img class="push_up_img" src="/img/22917-pushup 1.png" alt="push-up" />
+      <div class="several_box">
+        <h2 class="attetion">최고 기록</h2>
+        <h1 class="several_time">75회</h1>
+      </div>
+      <button class="push_up_button" onclick="startTimer()">완료</button>
```

Comment:
```suggestion
      <button class="push_up_button">버튼</button>
```
동작하지 않는 자바스크립트 코드는 지워주세요

Reply:
수정했습니다.